pkgs/development/tools/protoc-gen-twirp_typescript/default.nix

```diff
@@ -1,24 +1,24 @@
 { lib, buildGoModule, fetchFromGitHub }:
 
-buildGoModule {
+buildGoModule rec {
   pname = "protoc-gen-twirp_typescript";
-  version = "unstable-2021-03-29";
+  version = "535986b31881a214db3c04f122bcc96a2823a155"; # last released version before archive by repo owner
 
   src = fetchFromGitHub {
     owner = "larrymyers";
     repo = "protoc-gen-twirp_typescript";
-    rev = "97fd63e543beb2d9f6a90ff894981affe0f2faf1";
+    rev = "${version}";
     sha256 = "sha256-LfF/n96LwRX8aoPHzCRI/QbDmZR9yMhE5yGhFAqa8nA=";
   };
 
-  vendorSha256 = "sha256-WISWuq1neVX4xQkoamc6FznZahOQHwgkYmERJF40OFQ=";
-
-  subPackages = [ "." ];
```

Comment:
Please keep `subPackages = [ "." ];`, since otherwise the example https://github.com/larrymyers/protoc-gen-twirp_typescript/tree/master/example/cmd/haberdasher will be put into the bin-output:

```
results/protoc-gen-twirp_typescript/
└── bin
    ├── haberdasher
    └── protoc-gen-twirp_typescript
```

Reply:
done